src/services/style-service.ts

```diff
@@ -86,142 +97,144 @@ const resolveTextDecoration = (fontStyleOptions: FontStyleOptions[] | undefined)
  */
 const createStyleService = (theme: ExtendedTheme, layoutService: LayoutService): StyleService => {
   const chartStyling = layoutService.layout.components?.find((n) => n.key === "theme");
-  const headerStyling = chartStyling?.[Path.Header];
-  const dimensionValue = chartStyling?.[Path.DimensionValues];
-  const measureValueStyling = chartStyling?.[Path.MeasureValues];
-  const measureLabelStyling = chartStyling?.[Path.MeasureLabels];
-  const totalValuesStyling = chartStyling?.[Path.TotalValues];
-  const nullValueStyling = chartStyling?.[Path.NullValues];
-  const gridStyling = chartStyling?.[Path.Grid];
-  const getThemeStyle = (paths: string[], attribute: string) => theme.getStyle(BASE_PATH, paths.join("."), attribute);
+  const headerStyling = chartStyling?.header;
+  const dimensionValue = chartStyling?.dimensionValues;
+  const measureValueStyling = chartStyling?.measureValues;
+  const measureLabelStyling = chartStyling?.measureLabels;
+  const totalValuesStyling = chartStyling?.totalValues;
+  const nullValueStyling = chartStyling?.nullValues;
+  const gridStyling = chartStyling?.grid;
+
+  const getThemeStyle = (basePath: ThemeBasePath, path: ThemePath, attribute: ThemeAttribute) =>
```

Comment:
By using a specific combination of `basePath`, `path` and `attribite`, the fallback values in the theme can be used.